### DISCUSSION TITLE
Scrum Prep Milestone 2-5

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#156] [General]
Each team member will comment progress made through the past week.

**Comment by JadStelar** [comment]
Worked on the following sections of the Milestone 2 document: Informative part (1), interface requirements (2.2.4), and concept analysis (3.1).
-Jahdiel Montero

**Comment by Cabinetman** [comment]
This week, I focused on addressing milestone 1 feedback by ensuring grammatical consistency throughout the project documentation. Additionally, I completed a lecture topic task on algebra by functions for the implementation of Mercado Colegial.

**Comment by ActuallySeb** [comment]
Over the last couple of weeks I've been working on the implementation for Docker, which has far more taxing than originally anticipated. I couldn't even do the full implementation to include the database image as I'm already having enough trouble as is with the front end image.

**Comment by JuanIranzo** [comment]
Created and added sequence diagrams to section 3.1.1 of the Milestone 2 documentation. Succesfully implemented the controller for the Review entity of the database, with functional CRUD operations.

**Comment by CovasJJ** [comment]
This week I worked on various tasks involved with the documentation. Helped revamp section 1 Informative Part. Fixed an error in section 2.1.5, added new terminology in section 2.1.2 Terminology, and worked on revising section 3.1 Concept Analysis.

**Comment by AlvinBadillo** [comment]
This week I implemented the model of the review entity in the backend of the project and I also completed a sequence diagram in the Milestone 2 documentation in section 3.1.1.

**Comment by FabiolaZTorres** [comment]
I reviewed some tasks so they could be closed and merged the `Website-Frontend` branch into `main`. 

-Fabiola Z. Torres Maldonado

**Comment by franrov4** [comment]
Conducted a thorough review of the tasks assigned to the database team. I successfully integrated the database with our project's backend, updated the item model, and established a new server-side route for this model.

**Comment by JoshDG03** [comment]
worked on updating the documentation for this milestones LTT, implemented software design, domain engineering and requirements concepts to the 1.3 section

**Comment by JuanDelPueblo** [comment]
Created section 2.2.6 Engineering Requirements of the Milestone 2 documentation focusing on feature prioritization, use cases, and different modeling techniques.

**Comment by cristian-barreras** [comment]
I revised the section 2.1.1 of the documentation.

**Comment by YampierFernandez** [comment]
Reviewed tasks for database team, participated in task for Statecharts of the shopping journey.

**Comment by jonael-garcia26** [comment]
This week I started the email confirmation task with @F3rnand0hr  We created a file to handle the email sending, where a token should be generated for each user, and an email template is sent from the account mercado.colegial802@gmail.com. We had trouble fully implementing this feature because we need the database to be more complete.

**Comment by Christian2101** [comment]
Worked in the documentation. Will continue to make sure al changes made are within the specified format until the deadline of the project. project

**Comment by elliot-gomez** [comment]
This week, I integrated revised sections 1.2.2, 1.3, 1.4, 1.5, and 2.1.4 from the Milestone 1 Documentation into the Milestone 2 Documentation. I ensured all changes were recorded in the change log and used appropriate highlighting to show additions, edits, and removals. Additionally, I made general wording adjustments to improve clarity and align with the professor's feedback. I also worked on revamping Section 2, "Descriptive Part," by incorporating lecture concepts.

- Elliot J. Gomez Coriano

**Comment by gcal77** [comment]
This week I worked on revising section 2.2.3 of the documentation. Additionally, I am currently working on adding algebras into the documentation.

-Gabriel Calvente

**Comment by Kaariinaa08** [comment]
This week, I revised Section 2.1.5 of the documentation per professor feedback, incorporating concepts from the "Phenomena and Concepts" lecture, and created a log of changes for milestone one. Additionally, I developed Section 2.4 on "Validation and Verification" for the Descriptive Part of the document, guided by the lecture on "Documents."

- Karina Lopez Rodriguez
